toolbox/linux/which.rst

```diff
@@ -0,0 +1,111 @@
+Which Linux Should I Use?
+=========================
+
+Unlike Windows or OS X, there are several different "flavors" of Linux you can
+use. We call them "`Linux distributions
+<https://en.wikipedia.org/wiki/Linux_distribution>`_."
+
+Distributions?
+--------------
+
+An operating system is made up of many different pieces that come together to
+form a whole. There's the "kernel" (the core of the operating system), which is
+always Linux. But there are lots of other different parts, like the user
+interface, software for managing network connections, etc.
+
+A Linux distribution simply chooses all of the pieces required to form a usable
+operating system. The distribution glues everything together, making sure
+everything works properly.
+
+Linux distributions also provide ways for you to install software. On OS X or
+Windows, when you need software you probably search the web, download an
+installer, and run it. Linux distributions, on the other hand, make their own
+"installers" for most programs you might want to use. They usually have "package
+managers" that let you install these programs with a single command. This is
+typically easier, safer, and faster than installing (and updating) software on
+OS X or Windows.
+
+There are tons of choices of distribution, with names like Ubuntu, Debian, Mint,
+Fedora, Arch Linux, and Gentoo (to name a few). With all these choices, how do
+you know which one to use?
+
+**If it's your first time using Linux, use Ubuntu.** It's a fairly friendly
+place to start, and there's tons of help available for it. Check it out `here
+<http://www.ubuntu.com/>`_, or go straight to the `downloads page
+<http://www.ubuntu.com/download/desktop>`_ (you don't have to give them money,
```

Comment:
True, but I still think we should encourage charitable donations wherever we can.